packages/swapper/src/types.ts

```diff
@@ -389,12 +389,14 @@ export type Swap = {
   buyTxHash?: string
   statusMessage?: string | [string, Polyglot.InterpolationOptions] | undefined
   sellAccountId: AccountId
+  buyAccountId: AccountId | undefined
```

Comment:
@NeOMakinG ikik, but unless we want to go with a large refactor, think that's the perfect solution for the time being so we can get this shipped!